hw/ip/pattgen/data/pattgen_testplan.hjson

```diff
@@ -50,6 +50,43 @@
       milestone: V2
       tests: ["pattgen_perf"]
     }   
+    {
+      name: cnt_rollover
+      desc: '''
+            Checking ip operation with random counter values
+
+            Stimulus:
+              - Program the pre-divider and size registers to unconstraint random values
+              - Program the clk_cnt, bit_cnt and rep_cnt to values less than but close to
+		predivider and size registers, so that counting would take a reasonable
+		number of clock cycles
+	      - include programming for corner cases
+                repeat programming a random number of times
+              - Start and stop channels quickly
+              - Clear interrupts quickly
+            Checking:
+	      - Include functional cover point that rollover value is reached and counter is reset
+              - Ensure patterns are correctly generated
+              - Ensure interrupts are robust asserted and cleared (e.g. at the high data rate)
```

Comment:
misalign, are you using tab?

Reply:
I do not use tab since the number of spaces vary in tab. I aligned the lines

Reply:
the alignment seems still off
> 	      - include programming for corner cases
>                 repeat programming a random number of times
>               - Start and stop channels quickly
>               - Clear interrupts quickly
>             Checking:
> 	      - Include functional cover point that rollover value is reached and counter is reset
>               - Ensure patterns are correctly generated
>               - Ensure interrupts are robust asserted and cleared (e.g. at the high data rate)

Reply:
I agree.  Something is not right there.

Reply:
I aligned this section

Reply:
I aligned this section

Reply:
I just aligned and generated a document, and I view both test plan and document in front of me, while the lines in a test plan are aligned, the lines in document are still  misaligned. The document was successfully generated (built), committed and pushed to origin

Reply:
looks like you do use some tab. If you use vim/gvim as editor, you can search `\t` to find the tab